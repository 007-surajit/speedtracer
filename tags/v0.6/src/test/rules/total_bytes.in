hintlet_total_bytes.js

# Input Record 1
# Expected: none
# This will load a single network event with 200000 bytes of data.  The minor
# alarm triggers at 500,000 bytes
{"data":{"domain":"www.example.com","headers":{"Content-Length":"200000","User-Agent":"Mozilla/5.0 (Windows; U; Windows NT 5.1; en-US) AppleWebKit/530.11 (KHTML, like Gecko) Chrome/2.0.182.0 Safari/530.11","Content-Type":"image/jpeg"},"resourceId":"0NetworkResourceEvent99","url":"http://www.example.com/5608_1_poguepalmpre_75x50.jpg"},"time":14999.004000008106,"type":23}

# Input Record 2
# Expected: none
# This will load a single network event with 200000 bytes of data.  The minor
# alarm triggers at 500,000 bytes
{"data":{"domain":"www.example.com","headers":{"Content-Length":"200000","User-Agent":"Mozilla/5.0 (Windows; U; Windows NT 5.1; en-US) AppleWebKit/530.11 (KHTML, like Gecko) Chrome/2.0.182.0 Safari/530.11","Content-Type":"image/jpeg"},"resourceId":"0NetworkResourceEvent99","url":"http://www.example.com/5608_1_poguepalmpre_75x50.jpg"},"time":14999.004000008106,"type":23}

# Input Record 3
# Expected: HINTLET RECORD
# This record should put us past the 500,000 byte threshold
{"data":{"domain":"www.example.com","headers":{"Content-Length":"200000","User-Agent":"Mozilla/5.0 (Windows; U; Windows NT 5.1; en-US) AppleWebKit/530.11 (KHTML, like Gecko) Chrome/2.0.182.0 Safari/530.11","Content-Type":"image/jpeg"},"resourceId":"0NetworkResourceEvent99","url":"http://www.example.com/5608_1_poguepalmpre_75x50.jpg"},"time":14999.004000008106,"type":23}

# Input Record 4
# Expected: none
# This record should put us at the 1,600,000 bytes.  next Threshold is 
#  2,000,000 bytes.
{"data":{"domain":"www.example.com","headers":{"Content-Length":"1000000","User-Agent":"Mozilla/5.0 (Windows; U; Windows NT 5.1; en-US) AppleWebKit/530.11 (KHTML, like Gecko) Chrome/2.0.182.0 Safari/530.11","Content-Type":"image/jpeg"},"resourceId":"0NetworkResourceEvent99","url":"http://www.example.com/5608_1_poguepalmpre_75x50.jpg"},"time":14999.004000008106,"type":23}

# Input Record 5
# Expected: HINTLET_RECORD
# This record should put us past the 2,000,000 byte threshold
{"data":{"domain":"www.example.com","headers":{"Content-Length":"1000000","User-Agent":"Mozilla/5.0 (Windows; U; Windows NT 5.1; en-US) AppleWebKit/530.11 (KHTML, like Gecko) Chrome/2.0.182.0 Safari/530.11","Content-Type":"image/jpeg"},"resourceId":"0NetworkResourceEvent99","url":"http://www.example.com/5608_1_poguepalmpre_75x50.jpg"},"time":14999.004000008106,"type":23}

# Input Record 6
# Expected: HINTLET_RECORD
# This record should put us past the 10,000,000 byte threshold
{"data":{"domain":"www.example.com","headers":{"Content-Length":"60000000","User-Agent":"Mozilla/5.0 (Windows; U; Windows NT 5.1; en-US) AppleWebKit/530.11 (KHTML, like Gecko) Chrome/2.0.182.0 Safari/530.11","Content-Type":"image/jpeg"},"resourceId":"0NetworkResourceEvent99","url":"http://www.example.com/5608_1_poguepalmpre_75x50.jpg"},"time":14999.004000008106,"type":23}

# Input Record 7
# Expected: none
# This page transition should reset the byte counts
{"data":{"url":"http://example.com"},"type":16}

# Input Record 8
# Expected: HINTLET_RECORD
# This record should put us past the 500,000 byte threshold
{"data":{"domain":"www.example.com","headers":{"Content-Length":"700000","User-Agent":"Mozilla/5.0 (Windows; U; Windows NT 5.1; en-US) AppleWebKit/530.11 (KHTML, like Gecko) Chrome/2.0.182.0 Safari/530.11","Content-Type":"image/jpeg"},"resourceId":"0NetworkResourceEvent99","url":"http://www.example.com/5608_1_poguepalmpre_75x50.jpg"},"time":14999.004000008106,"type":23}

