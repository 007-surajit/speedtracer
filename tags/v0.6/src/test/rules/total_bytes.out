{"hintletRule":"Total Bytes Downloaded","timestamp":14999.004000008106,"description":"More than 500000 bytes downloaded.","refRecord":3,"severity":3}
{"hintletRule":"Total Bytes Downloaded","timestamp":14999.004000008106,"description":"More than 2000000 bytes downloaded.","refRecord":5,"severity":2}
{"hintletRule":"Total Bytes Downloaded","timestamp":14999.004000008106,"description":"More than 10000000 bytes downloaded.","refRecord":6,"severity":1}
{"hintletRule":"Total Bytes Downloaded","timestamp":14999.004000008106,"description":"More than 500000 bytes downloaded.","refRecord":8,"severity":3}
